backend/pkg/api/data_access/vdb_blocks.go

```diff
@@ -361,37 +361,22 @@ func (d *DataAccessService) GetValidatorDashboardBlocks(ctx context.Context, das
 		Slot     uint64              `db:"slot"`
 		ClReward decimal.NullDecimal `db:"cl_reward"`
 	}{}
-	if utils.Config.Chain.ClConfig.DepositChainID == 17000 {
-		clRewardsQuery := goqu.Dialect("postgres").
-			From(goqu.T("consensus_payloads")).
-			Select(
-				goqu.C("slot"),
-				goqu.L("cl_attestations_reward / 1e9 + cl_sync_aggregate_reward / 1e9 + cl_slashing_inclusion_reward / 1e9 AS cl_reward"),
-			).Where(goqu.C("slot").In(slots))
-		clRewardsQuerySql, args, err := clRewardsQuery.Prepared(true).ToSQL()
-		if err != nil {
-			return nil, nil, err
-		}
-		err = d.alloyReader.SelectContext(ctx, &clRewardsData, clRewardsQuerySql, args...)
-		if err != nil {
-			return nil, nil, err
-		}
-	} else {
-		clRewardsQuery := goqu.Dialect("postgres").
-			From(goqu.L("mainnet.validator_proposal_rewards_slot")).
-			Select(
-				goqu.C("slot"),
-				goqu.L("attestations_reward / 1e9 + sync_aggregate_reward / 1e9 + slasher_reward / 1e9 AS cl_reward"),
-			).Where(goqu.C("slot").In(slots))
-		clRewardsQuerySql, args, err := clRewardsQuery.Prepared(true).ToSQL()
-		if err != nil {
-			return nil, nil, err
-		}
-		err = d.clickhouseReader.SelectContext(ctx, &clRewardsData, clRewardsQuerySql, args...)
-		if err != nil {
-			return nil, nil, err
-		}
+
+	clRewardsQuery := goqu.Dialect("postgres").
+		From(goqu.L("validator_proposal_rewards_slot")).
```

Comment:
That table doesn't seem to exist yet on clickhouse for testnets. @invis-bitfly Are there missing migrations files or something? Otherwise we might need to keep using alloy data for testnets